etc/notes/CHANGES_7.0.0.md

```diff
@@ -0,0 +1,248 @@
+
+### `@aws-sdk/credential-providers` is now required for MONGODB-AWS authentication
+
+Previous versions of the driver contained two implementations for AWS authentication and could run the risk of the custom driver implementation not supporting all AWS authentication features as well as not being correct when AWS makes changes. Using the official AWS SDK in all cases alleviates these issues.
+
+```bash
+npm install @aws-sdk/credential-providers
+```
+
+### Custom AWS credential provider takes highest precedence
+
+When providing a custom AWS credential provider via the auth mechanism property `AWS_CREDENTIAL_PROVIDER`, it will now take the highest precedence over any other AWS auth method.
+
+### Explicitly provided credentials no longer accepted with MONGODB-AWS authentication
+
+AWS environments (such as AWS Lambda) do not have credentials that are permanent and expire within a set amount of time. Providing credentials in the URI or options would mandate that those credentials would be valid for the life of the `MongoClient`, which is problematic. With this change, the installed required AWS SDK will now fetch credentials using the environment, endpoints, or a custom credential provider.
+
+This means that for AWS authentication, all client URIs MUST now be specified as:
+
+```ts
+import { MongoClient } from 'mongodb';
+
+const client = new MongoClient('mongodb<+srv>://<host>:<port>/?authMechanism=MONGODB-AWS');
+```
+The previous method of providing URI encoded credentials based on the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` directly in the connection string will no longer work.
+## ⚙️ Error handling improvements
+
+### Dropping a collection returns false instead of throwing when NS not found
+
+This change has been made for consistency with the common drivers specifications.
+
+### Aggregate with write concern and explain no longer throws client-side
+
+This will now throw a `MongoServerError` instead.
+
+### All encryption-related errors now subclass MongoError
+
+The driver aims to ensure that all errors it throws are subclasses of `MongoError`. However, when using CSFLE or QE, the driver's encryption implementation could sometimes throw errors that were not instances of `MongoError`.
+
+Now, all errors thrown during encryption are subclasses of `MongoError`.
+
+### 'PoolRequstedRetry' error label renamed to 'PoolRequestedRetry'
+
+The `PoolClearedError` thrown in cases where the connection pool was cleared now fixes the typo in the error label.
+
+## 💥 Misc breaking improvements
+
+### Change streams no longer whitelist `$changeStream` stage options
+
+Uses are now able to pass any option to `collection.watch()` and if it is invalid in the `$changeStream` stage of the pipeline the server will error. This is to allow users to provide newly added options quickly that are not in our public types.
```

Comment:
```suggestion
Uses are now able to pass any option to `collection.watch()` and if it is invalid in the `$changeStream` stage of the pipeline the server will error.
```

Minor, but we really only intend for internal users to encounter this through mongosh, right?  Suggest removing all mention of using unsupported options.

Reply:
I adjusted the wording, I think it's still worthwhile to call out motivation for changes in our release notes; in this case, unknown to the driver and unsupported aren't the same thing